mini-e2e-helm.groovy

```diff
@@ -128,6 +128,16 @@ node('cico-workspace') {
 			ssh 'cd /opt/build/go/src/github.com/ceph/ceph-csi && make containerized-build CONTAINER_CMD=podman TARGET=e2e.test'
 		}
 		stage("deploy k8s-${k8s_version} and rook") {
+			def rook_version = sh(
```

Comment:
do we need podman log in this stage?

Reply:
`podman_login()` was called in a previous stage, so the credentials in the auth-cache are still there.